control-plane/pkg/reconciler/consumer/controller.go

```diff
@@ -90,7 +90,7 @@ func NewController(ctx context.Context, watcher configmap.Watcher) *controller.I
 		impl.GlobalResync(consumerInformer.Informer())
 	}
 
-	cgreconciler.ResyncOnStatefulSetChange(ctx, globalResync)
+	cgreconciler.ResyncOnStatefulSetChange(ctx, impl.FilteredGlobalResync, consumerInformer.Informer())
```

Comment:
The provided informer here will have consumers but in the function we do:

```golang
		filteredResync(func(i interface{}) bool {
			cg, ok := i.(*kafkainternals.ConsumerGroup)
			if !ok {
				return false
			}
```

Reply:
that makes sense, thanks for catching this! (just saved me lots of debugging time)